analytics/cdk/stream-emr-on-eks/source/lib/emr_on_ec2_stack.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, scope: Construct, id: str, emr_version: str, cluster_name:str
 
         # emr service role
         svc_role = iam.Role(self,"EMRSVCRole",
-            assumed_by=iam.ServicePrincipal("elasticmapreduce.amazonaws.com"),
+            assumed_by=iam.ServicePrincipal("eks.amazonaws.com"),
```

Comment:
This IAM update is a bit confusing for me. This file looks to be setting up an EMR on EC2 cluster, are you sure we want the EKS service for the principal vs elasticmapreduce ?